client/wc-api/user/selectors.js

```diff
@@ -8,7 +8,8 @@ import { DEFAULT_REQUIREMENT } from '../constants';
 const getCurrentUserData = ( getResource, requireResource ) => (
 	requirement = DEFAULT_REQUIREMENT
 ) => {
-	return requireResource( requirement, 'current-user-data' ).data || {};
+	const initalState = wcAdminServer.currentUserData; // eslint-disable-line
```

Comment:
Not a big deal, but what do you think about only disabling `no-undef` rule, so other eslint rules will still be applied to that line:
```suggestion
	const initalState = wcAdminServer.currentUserData; // eslint-disable-line no-undef
```